mmdet/models/roi_heads/bbox_heads/bbox_head.py

```diff
@@ -108,8 +108,12 @@ def custom_accuracy(self):
     @auto_fp16()
     def forward(self, x):
         if self.with_avg_pool:
-            x = self.avg_pool(x)
-        x = x.view(x.size(0), -1)
+            if x.shape[0] > 0:
+                x = self.avg_pool(x)
```

Comment:
1. May indicate why we need to do that, e.g., avg_pool does not support empty tensor 
2. may use x.numel() rather than x.shape[0] for consistency